reconcile/utils/terraform_client.py

```diff
@@ -511,38 +457,51 @@ def format_output(self, output, type):
             return data[list(data.keys())[0]]
         return data
 
-    def construct_oc_resource(self, name, data, account, annotations):
-        body = {
-            "apiVersion": "v1",
-            "kind": "Secret",
-            "type": "Opaque",
-            "metadata": {
-                "name": name,
-                "annotations": {
-                    "qontract.recycle": "true"
-                }
-            },
-            "data": {}
-        }
+    def populate_terraform_output_secrets(self, resource_specs: TerraformResourceSpecDict, init_rds_replica_source: bool = False) -> None:
```

Comment:
How come pylint or Black arent whining about the length of this 137-character line?

Reply:
i shortened the line